transmartApp/src/main/groovy/org/transmart/authorization/QueriesResourceAuthorizationDecorator.groovy

```diff
@@ -64,6 +65,14 @@ class QueriesResourceAuthorizationDecorator
         res
     }
 
+    @Override
+    QueryResult runDisablingQuery(Long id,
```

Comment:
What does this method do? Disable a query? Then it should not return a result, right?